src/writer/follow.ts

```diff
@@ -1,6 +1,21 @@
 import { FOLLOWS_LIMIT_PER_USER } from '../helpers/limits';
 import db from '../helpers/mysql';
 
+const NETWORK_WHITELIST = [
+  's',
+  's-tn',
+  'eth',
+  'matic',
+  'arb1',
+  'oeth',
+  'gor',
+  'sep',
+  'linea-testnet',
+  'sn',
+  'sn-tn',
+  'sn-sep'
+];
```

Comment:
Maybe we can ignore `s-tn` ?

Reply:
Not sure if we want to have separate follow for every evm network, like someone with same address on `eth` `matic` `arb1` etc have to follow on each network?

Reply:
This is the network of the space, and not the voter

Reply:
Will make a different list for testnet and mainnet